### PR TITLE
Fix for redesigned CI/CD pipeline failure

### PR DESCRIPTION
**Pull request by MukuFlash03**
**Failed** [workflow run](https://github.com/e-mission/em-public-dashboard/pull/125#issuecomment-2289651090)

Changes:
Adding suffix to image tag to match image name in docker-compose.yml

Details in these comments ([link](https://github.com/e-mission/em-public-dashboard/pull/125#issuecomment-2289658512), [link](https://github.com/e-mission/em-public-dashboard/pull/125#issuecomment-2289670640))

### CODE REVIEW

**Comment by shankari** [comment]
@MukuFlash03 waiting for the "two tag upload" before I review and merge

**Comment by MukuFlash03** [comment]
Almost done, running some CI/CD tests to ensure it works.

**Comment by MukuFlash03** [comment]
Added the changes. Details on successful testing in [this comment](https://github.com/e-mission/em-public-dashboard/pull/125#issuecomment-2290160898) in the original PR.

File: .env.repoTags
Comment (shankari) [inline .env.repoTags:1]: I am not going to insist on this right now, but as part of https://github.com/e-mission/e-mission-docs/issues/1082, we should consider putting the notebook tag in here as well, and just using this instead of the artifacts

File: .github/workflows/image_build_push.yml
Comment (shankari) [inline .github/workflows/image_build_push.yml:37]: I am not going to insist on this, but we should revisit this as part of 
https://github.com/e-mission/e-mission-docs/issues/1082

" Consider switching from $GITHUB_ENV to step outputs. I think that the step outputs are the recommended approach to pass values from one step to another, but we should verify"

**Review by shankari — APPROVED** [review]

**Pull request merged**
